Src/Extern/HYPRE/AMReX_HypreABecLap.cpp

```diff
@@ -259,19 +259,35 @@ HypreABecLap::loadVectors (MultiFab& soln, const MultiFab& rhs)
     soln.setVal(0.0);
 
     MultiFab rhs_diag(rhs.boxArray(), rhs.DistributionMap(), 1, 0);
+
+#ifdef AMREX_USE_GPU
+    if (Gpu::inLaunchRegion() && rhs_diag.isFusingCandidate()) {
+        auto const& rhs_diag_ma = rhs_diag.arrays();
+        auto const& rhs_ma = rhs.const_arrays();
+        auto const& diaginv_ma = diaginv.const_arrays();
+        ParallelFor(rhs_diag,
+        [=] AMREX_GPU_DEVICE (int box_no, int i, int j, int k) noexcept
+        {
+            rhs_diag_ma[box_no](i,j,k) = rhs_ma[box_no](i,j,k) * diaginv_ma[box_no](i,j,k);
+        });
+        Gpu::streamSynchronize();
+    } else
+#endif
+    {
 #ifdef AMREX_USE_OMP
 #pragma omp paralle if (Gpu::notInLaunchRegion())
```

Comment:
#pragma omp paralle if (Gpu::notInLaunchRegion())

parallel is misspelled. And this is sprinkled throughout the area, no doubt copy-pasted. (I guess OMP hasn't been turned on in this section in a while?)

Reply:
Thanks for catching this!